### PR TITLE
Merge in versioned code

### PR DESCRIPTION
**Pull request by kevindew**
Trello: https://trello.com/c/UdrY0K9l/574-migrate-to-versioned-code-database

This merges the code under the Versioned namespace into the main namespace of the application to remove it.

### CODE REVIEW

File: app/models/versioning/edition.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module Versioning
+  class Edition < ApplicationRecord
+    self.table_name = "versioned_editions"
+
+    attr_readonly :number, :document_id
+
+    # rubocop:disable Rails/InverseOf
```
Comment (1pretz1) [inline app/models/versioning/edition.rb]: Why not use inverse_of here?

File: app/services/preview_service.rb [hunk 2]
```diff
@@ -1,72 +1,61 @@
...
 
-  def try_create_preview(args)
-    save_changes(args)
-    try_publish_draft(document) unless has_issues?
+  def try_create_preview
+    if has_issues?
+      edition.update!(revision_synced: false)
+    else
+      begin
+        upload_assets(edition)
```
Comment (emmabeynon) [inline app/services/preview_service.rb]: Just a small one - can we just call `create_preview` here instead of repeating these lines?

Comment (kevindew) [reply app/services/preview_service.rb]: Yup - good catch

Comment (kevindew) [reply app/services/preview_service.rb]: @emmabeynon refactored in 5a4d96cdbc4088848ffb97db4123b67e8f50debe

**Pull request merged**